compiler/src/test/scala/org/scalajs/core/compiler/test/JSExportTest.scala

```diff
@@ -17,6 +17,8 @@ import org.junit.Test
 
 import org.junit.Assume._
 
+import scala.language.postfixOps
```

Comment:
Just write the dots out :)

Reply:
Removed in https://github.com/scala-js/scala-js/pull/3633/commits/090dc49f91c691ab60c8977431fc5f54a63d6cf2